modules/oneVideoBidAdapter.md

```diff
@@ -298,3 +298,110 @@ var adUnits = [
       }
   ]
 ```
+# Content Object Support
+The oneVideoBidAdapter supports passing of OpenRTB V2.5 Content Object.
+
+```
+const adUnits = [{
+            code: 'video1',
+            mediaTypes: {
+                video: {
+                    context: 'outstream',
+                    playerSize: [640, 480]
+                }
+            },
+            bids: [{
+                bidder: 'oneVideo',
+                params: {
+                    video: {
+                        playerWidth: 640,
+                        playerHeight: 480,
+                        mimes: ['video/mp4', 'application/javascript'],
+                        protocols: [2, 5],
+                        api: [1, 2],
+                        ttl: 300,
+                        content: {
+                            id: "1234",
+                            title: "Title",
+                            series: "Series",
+                            season: "Season",
+                            episode: "Episode",
+                            artist: "Artist",
+                            cat: [
+                                "IAB1",
+                                "IAB1-1",
+                                "IAB1-2",
+                                "IAB2",
+                                "IAB2-1"
+                            ],
+                            genre: "Genre",
+                            album: "Album",
+                            isrc: "Isrc",
```

Comment:
`contentStringKeys` array @ line 323 in adapter doesn't have many strings like `iSrc` , `producer`, `url`... is it intended?

Reply:
Hi @kprasadpvv, thanks for catching.
In my first iteration I supported all ORTB 2.5 content object params but after validating with Vinay I found that the ad-server does not support all of them so I reduced the functionality to match.
I have removed the redundant params from the example accordingly.
Thank you Sir